platform/linux/mod.rs

```diff
@@ -28,6 +28,10 @@ const MAX_FDS_IN_CMSG: u32 = 64;
 // Yes, really!
 const MAP_FAILED: *mut u8 = (!0usize) as *mut u8;
 
+// The value Linux returns for SO_SNDBUF
+// is not the size we are actually allowed to use...
+const RESERVED_SIZE: usize = 256;
```

Comment:
Well, I took it from your original code -- so it's up to you to explain it :-P

I thought that it might have been related to `MAX_FDS_IN_CMSG`, which would match the size -- but then, you were also deducting space depending on the number of FDs *actually* sent; so that doesn't really make sense either...

Anyway, as you have seen, I changed in the next commit.